apps/host/src/features/notice/use-notice-form.ts

```
@@ -0,0 +1,152 @@
+      if (pinnedCount >= 3) {
+        toast.show(
+          '상단 고정할 수 있는 공지 수를 초과했어요.',
+          '기존 공지를 고정 해제한 후 시도해주세요.',
+        );
+        return;
+      }
+    }
+
+    const newImages: File[] = [];
+    const keepImageUrls: string[] = [];
+    images.forEach((img) =>
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)